lua/astrocommunity/snippet/mini-snippets/init.lua

```diff
@@ -0,0 +1,104 @@
+return {
+  "echasnovski/mini.snippets",
+  dependencies = "rafamadriz/friendly-snippets",
+  opts_extend = { "snippets" },
+  opts = function(_, opts)
+    local mini_snippets = require "mini.snippets"
+    local gen_loader = mini_snippets.gen_loader
+    opts.snippets = vim.list_extend(opts.snippets or {}, {
```

Comment:
I *think* the order here should be different: first "default" locations and then the ones from the `opts`. This way snippets read from the user supplied locations will be preferred if there are snippets with the same prefix ("last one wins"). This should allow users to override snippets from "default" locations.

Reply:
Ah we can flip that! The way these are imported in AstroNvim should make this a non-issue because all community specs are imported before the user's code so all of them should be added after